### PR TITLE
Include current count error message

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: temba/api/v1/serializers.py [hunk 1]
```diff
@@ -446,10 +446,12 @@ def validate(self, data):
             if not ContactField.is_valid_key(key):
                 raise serializers.ValidationError(_("Generated key for '%s' is invalid or a reserved name") % label)
 
-        if not self.instance and ContactField.objects.filter(org=self.org).count() >= ContactField.MAX_ORG_CONTACTFIELDS:
-            raise serializers.ValidationError('You have reached %s contact fields, '
+        fields_count = ContactField.objects.filter(org=self.org).count()
+        if not self.instance and fields_count >= ContactField.MAX_ORG_CONTACTFIELDS:
+            raise serializers.ValidationError('You have reached %s contact fields (%s contact fields currently), '
```
Comment (rowanseymour) [inline temba/api/v1/serializers.py]: I think we can phrase this better, e.g.

_"This org has 267 contact fields and the limit is 250. You must delete existing ones before you can create new ones."_

Comment (norkans7) [reply temba/api/v1/serializers.py]: 👍

File: temba/contacts/views.py [hunk 2]
```diff
@@ -585,10 +587,11 @@ def clean_csv_file(self):
                 return self.cleaned_data['csv_file']
 
             def clean(self):
-                if ContactGroup.user_groups.filter(org=self.org).count() >= ContactGroup.MAX_ORG_CONTACTGROUPS:
-                    raise forms.ValidationError('You have reached %s contact groups, please remove some contact groups '
-                                                'to be able to import contacts in a contact group' %
-                                                ContactGroup.MAX_ORG_CONTACTGROUPS)
+                groups_count = ContactGroup.user_groups.filter(org=self.org).count()
+                if groups_count >= ContactGroup.MAX_ORG_CONTACTGROUPS:
+                    raise forms.ValidationError("This org has %s groups and the limit is %s. "
```
Comment (rowanseymour) [inline temba/contacts/views.py]: should be localized, i.e. `_("...")`

**Comment by nicpottier** [comment]
Ya I don't think we should be blowing up at the model level, but instead trying to catch things when users are taking particular actions.

**Comment by norkans7** [comment]
I think those sentry errors are because we used to create a group if does not exist from a flow definition when building it or when importing flows.
Do you mean we keep creating those until the user try to import contact or create one the view??

**Comment by nicpottier** [comment]
Yes, I think we should only be refusing to create groups when they explicitly are creating groups through the UI. I suppose I could also see us refusing to do an import until they remove some groups.

But actions such as importing flows, creating groups in flows etc, should succeed even if over the limit.

**Comment by norkans7** [comment]
Right that makes sense

**Pull request merged**